neverwinter/nwscript/native/scriptcompparsetree.cpp

```diff
@@ -416,15 +418,13 @@ int32_t CScriptCompiler::GenerateParseTree()
 
 	while (bAlwaysTrue)
 	{
-		// Remove the topmost state from the stack.
```

Comment:
nit: Certainly a useless comment, but why remove in this change?

Reply:
I had hundreds of comments in the code to keep my mind straight in this quagmire.  I went through a deleted them all just before I submitted the PR and I probably thought this was one of mine.  Oopsie. Should be returned in a1f56c32.